### ISSUE TITLE
global module location

### ISSUE THREAD

**Issue by domtra** [isaacs/nave#96]
Hi there. This is more a question rather than an issue.

Is it intended, that when you only switch minor or patch versions of node, that you need to reinstall all global npm modules? 

For me it used to be different before I wiped and re-setup my machine. Now, let's say I am using v8.14.0 and install a global package it will correctly make the binaries available in my PATH. However, when I update node to v8.14.1, these binaries will not be available anymore. I have to reinstall the package globally again.

It would be really nice, if at least patch versions would share a global directory. 

I am using zsh and `nave use x` will add the following env variables:

```shell
NAVE=8.14.0
NAVELVL=1
NAVENAME=8.14.0
NAVEPATH=/Users/xxx/.nave/installed/8.14.0/bin
NAVEVERSION=8.14.0
NAVE_DIR=/Users/xxx/.nave
NAVE_LOGIN=1
NAVE_ROOT=/Users/xxx/.nave/installed
NAVE_SRC=/Users/xxx/.nave/src
NODE_PATH=/Users/xxx/.nave/installed/8.14.0/lib/node
PATH=$PATH:/Users/xxx/.nave/installed/8.14.0/bin:/Users/xxx/bin
npm_config_binroot=/Users/xxx/.nave/installed/8.14.0/bin
npm_config_manroot=/Users/xxx/.nave/installed/8.14.0/share/man
npm_config_prefix=/Users/xxx/.nave/installed/8.14.0
npm_config_root=/Users/xxx/.nave/installed/8.14.0/lib/node
```

**Comment by ljharb** [comment]
Yes, because modules can be compiled for a specific version of node - global modules shouldn’t be shared across node versions.

**Comment by ukabu** [comment]
Any suggestion to manage the npm version? It would be really nice to be able to not only specify node version but also the npm version.
